src/chapters/orm/studio.rst

```diff
@@ -0,0 +1,114 @@
+Studio: Creating an ``AbstractEntity``
+======================================
+
+In :ref:`orm1-exercises`, you created a new persistent class, ``EventCategory``. We will eventually use this class to categorize events in our application. However, your application now has a fair amount of duplication across the two model classes. This studio has you reduce repetition (i.e. "DRY out your code") using inheritance and abstract classes.
+
+Getting Ready
+-------------
+
+In your ``coding-events`` repository, checkout out the `add-persistent-category branch <https://github.com/LaunchCodeEducation/coding-events/tree/add-persistent-category>`_ and create a new branch, ``abstract-entity-studio``, off of that branch.
+
+.. admonition:: Note
+
+   You can also branch off of the branch you created in the exercises, if you like. We have provided the branch mentioned to make sure you have good starter code for the studio, even if you didn't complete the exercises. 
+
+Your Tasks
+----------
+
+Let's look at both the ``Event`` and ``EventCategory`` classes. There are four class members that are (essentially) the same in both classes:
+
+- The ``id`` field
+- The ``getId`` method
+- The ``equals`` method
+- The ``hashCode`` method
```

Comment:
For accessibility, this should be an ``ol``.

Reply:
These elements don't have any necessary order, so `ul` elements are appropriate.